### PR TITLE
fixing CVE critical issues by resolving kerby/jline and wildfly libraries

### PR DESCRIPTION
**Pull request by xiangfu0**
1. Remove io.netty:netty library
2. Consolidate org.apache.kerby:kerb-core and org.apache.kerby:kerb-simplekdc version to 2.0.3
3. Consolidate org.jline:jline version to 3.22.0
4. Consolidate org.wildfly.common:wildfly-common version to 1.5.4.Final

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -857,6 +860,10 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <exclusion>
```
Comment (Jackie-Jiang) [inline pom.xml:863]: Do we need to exclude netty? We have netty-bom in the dependency

Comment (xiangfu0) [reply pom.xml:863]: We are using `io.netty:netty-all:jar:4.1.107.Final`, this is older dependency.

Comment (Jackie-Jiang) [reply pom.xml:863]: I don't follow. Since we have netty explicitly configured, it should pull the new netty, and there should be no need to exclude it

**Review by Jackie-Jiang — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Jackie-Jiang) [reply pom.xml:851]: Figured that netty is the old netty-all which is already deprecated, so it should be safe to exclude

**Pull request merged**